protocols/luci-proto-ipip/luasrc/model/network/proto_ipip.lua

```diff
@@ -0,0 +1,40 @@
+-- Copyright 2016 Roger Pueyo Centelles <roger.pueyo@guifi.net>
+-- Licensed to the public under the Apache License 2.0.
+
+local netmod = luci.model.network
+local interface = luci.model.network.interface
+local proto = netmod:register_protocol("ipip")
+
+function proto.get_i18n(self)
+	return luci.i18n.translate("IPv4-in-IPv4 (RFC2003)")
+end
+
+function proto.ifname(self)
+	return "ipip-" .. self.sid
+end
+
+function proto.opkg_package(self)
+	return "ipip"
+end
+
+function proto.is_installed(self)
+	return nixio.fs.access("/lib/netifd/proto/ipip.sh")
+end
```

Comment:
inside Makefile there is no dependency definition.
Which package install ipip.sh ?
If someone selects your LuCI package for installation it should also install the needed package for ipip protocol.
ipip protocol is not a standard protocol of OpenWrt build